systemjs/react-hello-world/index.html

```diff
@@ -0,0 +1,25 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <meta charset="utf-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <title>SystemJS AMD Modules from CDN Example</title>
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <script type="systemjs-importmap">
+    {
+      "imports": {
+        "react": "https://cdn.jsdelivr.net/npm/react/umd/react.production.min.js",
+        "react-dom": "https://cdn.jsdelivr.net/npm/react-dom/umd/react-dom.production.min.js",
+        "hello-world": "./dist/react-hello-world.js"
+      }
+    }
+  </script>
+  <script src="https://cdn.jsdelivr.net/npm/systemjs/dist/system.min.js"></script>
+  <script src="https://cdn.jsdelivr.net/npm/systemjs/dist/extras/amd.min.js"></script>
+  <script src="https://cdn.jsdelivr.net/npm/systemjs/dist/extras/use-default.min.js"></script>
+</head>
+<body>
+  <div id="react-root"></div>
+  <script type="systemjs-module" src="import:hello-world"></script>
```

Comment:
For this initial example it may make sense to just reference a normal relative URL to the app.

Alternatively if we really want to use an import map, then it should be a directory mapping like `app/` sort of thing perhaps.